### PR TITLE
docs: add templates for PR and issues

### LINKED ISSUE #32: New issue/PR template
We need a formal issue/PR template for people whom are going to create an issue/PR for KubeEdge.

xref: https://github.com/kubernetes/kubernetes/tree/master/.github/ISSUE_TEMPLATE

https://github.com/kubernetes/kubernetes/blob/master/.github/PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by rohitsardesai83**
this adds templates for reporting issues , enhancements , raising pull requests

Fixes #32

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+name: Bug Report
```
Comment (m1093782566) [inline .github/ISSUE_TEMPLATE/bug-report.md]: kind: bug-report?

kind: feature-request?

kind: question?

**Comment by rohitsardesai83** [comment]
@m1093782566 , @sids-b , please review. I have added the initial version of the templates to keep them simple. These templates will evolve and become better as we progress further. For now , the minimum fields would help ensure that we are following the contribution guidelines , providing enough info while raising bugs or reporting test failures and coming up with sufficient background info when submitting a new design proposal / enhancement. We can always revisit these checks later and come up with better ones.

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 1 above)
Comment (sids-b) [reply .github/ISSUE_TEMPLATE/bug-report.md]: @rohitsardesai83 i have added labels for the same

Comment (rohitsardesai83) [reply .github/ISSUE_TEMPLATE/bug-report.md]: I have added the kind labels for bugs , enhancements , pull requests etc.

Comment (m1093782566) [reply .github/ISSUE_TEMPLATE/bug-report.md]: I would suggest change `name` to `kind`. Thanks!

**Pull request merged**